### ISSUE TITLE
send command to tracker 

### ISSUE THREAD

**Issue by Elajaj** [tananaev/traccar#1851]
Hi tananaev, how can i send a GPRS command from tracker to tracker .
thank you.

**Comment by tananaev** [comment]
From tracker to tracker? I don't think it's possible. Why would you want to do that?

**Comment by Elajaj** [comment]
sorry  From **# traccar**  to tracker   :) 

i use MVT600 from meitrack, it can receive the GPRS commands, for exemple Enabling/Disabling the RFID Control:
GPRS Sending : @@T27,353358017784062,A17,1*D3\r\n 

thank you

**Comment by tananaev** [comment]
Traccar does support some commands for Meitrack. You can send them from web interface or API.

**Comment by Elajaj** [comment]
sorry but , How can i do it from traccar interface ?

**Comment by tananaev** [comment]
There is a button in devices panel:

![image](https://cloud.githubusercontent.com/assets/1161863/14310946/d69f0bc6-fc38-11e5-8622-1fbc501934a4.png)

**Comment by Elajaj** [comment]
Sorry but i need add more commands not exist  in traccar interface, it is not possible ?

**Comment by tananaev** [comment]
Yes, it is possible. Send me an email to discuss details. Address is in profile.
